docs/integrations/frontegg/sso-oidc.md

```diff
@@ -0,0 +1,172 @@
+By integrating Frontegg SSO with ngrok, you can:
+
+- **Restrict access to ngrok tunnels** only to users authenticated via Frontegg
+- **Use Frontegg security policies and MFA authenticators**.
+- **Use Frontegg's Dashboard to facilitate access to ngrok apps**.
+
+## Supported Features
+
+The ngrok integration with Frontegg supports:
+
+- **SP-Initiated SSO**: In this mode, users access ngrok edges and tunnels and are redirected to Frontegg for authentication.
+
+## Requirements
+
+To configure ngrok tunnels with Frontegg, you must have:
+
+- a Frontegg account with administrative rights to create apps
+- an [ngrok Enterprise Account](https://ngrok.com/pricing) with an authtoken or admin access to configure edges with OpenID Connect.
+
+## Configuration Steps
+
+To integrate ngrok with Frontegg SSO, you will need to:
+
+1. Configure Frontegg with the ngrok app
+1. Configure ngrok with the SSO settings provided by Frontegg
+
+## **Step 1**: Configure Frontegg {#configure-Frontegg}
+
+### Add the ngrok App in Frontegg
+
+1. Access the [Frontegg Portal](https://portal.frontegg.com/), and sign in using your Frontegg administrator account.
+
+1. On the left menu, click the environment you want to integrate with ngrok. In this example click the **Development** environment.
+
+1. On the left menu, click **Authentication**, click **Login method**, enter the URL `https://idp.ngrok.com/oauth2/callback` under the **Hosted login** field, and then click **Save**.
+   ![Frontegg configuration](img/ngrok_url_configuration_frontegg.png)
+
+1. On the left menu, click **Env settings**, and copy the value of both the **Client ID** and the **API key** fields.
+
+1. Click the **Domains** tab, and copy the value of the **Domain name**.
+
+
+### Grant access to Frontegg users
+
+Create a Frontegg user:
+
+1. On the left menu of the [Frontegg Portal](https://portal.frontegg.com/), click your environment, click **Backoffice**, click **Accounts**, and then click **Add New**.
+
+1. On the **Add new account** popup, enter a name for the account, `ngrok.com` in the **Domain** field, and then click **Save**.
+
+1. On the left menu, click **Backoffice**, click **Users**, and then click **Add New**.
+
+1. On the **Add new user** popup, enter a name, email, select the account you have created, and then click **Save**.
+   **Note**: By default, the user is assigned the **Admin** role.
+
+
+## **Step 2**: Configure ngrok {#configure-ngrok}
+
+ngrok can leverage Frontegg SSO in two ways:
+
+- From the ngrok CLI (using the `--oidc` parameter)
+- From the ngrok dashboard
+
+### **Option 1**: ngrok CLI
+
+> **Note:** For this tutorial, we assume you have an app running locally (i.e., on localhost:3000) with the ngrok client installed.
+
+1. Launch a terminal
+
+1. Enter the following command to launch an ngrok tunnel with Frontegg SSO:
+
+   ```bash
+   ngrok http 3000 --oidc=Frontegg_OAUTH_URL \
```

Comment:
the standard for env variables IS_ALL_CAPS. can we change this to FRONTEGG_OAUTH_URL? same with the client id and secret.

Reply:
You're right. Fixed.